.github/workflows/create_release.yml

```diff
@@ -48,16 +47,13 @@ jobs:
           GOPATH=$(go env GOPATH)
           ARCHIVE="elrond_""$APP_VER_SHORT""_""$GOOS""_""$GOARCH"".tgz"
           BUILD_DIR=${GITHUB_WORKSPACE}/build
-          ARWEN_VERSION=$(cat go.mod | grep arwen | sed 's/^.*arwen-wasm-vm *//' | tail -n 1)
-          ARWEN_PATH=${GITHUB_WORKSPACE}/build/arwen
-          WASMER_DIR=${GOPATH}/pkg/mod/github.com/\!elrond\!network/arwen-wasm-vm@${ARWEN_VERSION}/wasmer
+          WASM_VERSION=$(cat go.mod | grep wasm | sed 's/^.*wasm-vm-v1_4 *//' | tail -n 1)
+          WASMER_DIR=${GOPATH}/pkg/mod/github.com/\!elrond\!network/wasm-vm-v1_4@${WASM_VERSION}/wasmer
```

Comment:
do we still need this?

Reply:
yes, the libwasmer directory

Reply:
right, got confused was thinking about the arwen path

Reply:
This was the main reason the workflow failed. The following lines are fetched from this failed create release build flow:
https://github.com/ElrondNetwork/elrond-go/actions/runs/3655452536/jobs/6176794677

> ARWEN_PATH: /Users/runner/work/elrond-go/elrond-go/build/arwen
> WASMER_DIR: /Users/runner/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@/wasmer
> cp: /Users/runner/go/pkg/mod/github.com/!elrond!network/arwen-wasm-vm@/wasmer/libwasmer_darwin_amd64.dylib: > No such file or directory
> Error: Process completed with exit code 1.